content/accounts.md

```diff
@@ -0,0 +1,702 @@
+${url}
+If you didn't request this email, please ignore it.
+Thanks,
+The Meteor Todos team
+`
+  },
+  html(user, url) {
+    // This is where HTML email content would go.
+    // See the section about html emails below.
+  }
+};
+```
+
+As you can see, we can use the ES2015 template string functionality to generate a multi-line string that includes the password reset URL. We can also set a custom `from` address and email subject.
+
+<h4 id="html-emails">HTML emails</h4>
+
+If you've ever needed to deal with sending pretty HTML emails from an app, you know that it can quickly become a nightmare. Compatibility of popular email clients with basic HTML features like CSS is notoriously spotty, so it is hard to author something that works at all. Start with a [responsive email template](https://github.com/leemunroe/responsive-html-email-template) or [framework](http://foundation.zurb.com/emails/email-templates.html), and then use a tool to convert your email content into something that is compatible with all email clients. [This blog post by Mailgun covers some of the main issues with HTML email.](http://blog.mailgun.com/transactional-html-email-templates/)
+
+<h2 id="oauth">OAuth login</h2>
+
+In the distant past, it could have been a huge headache to get Facebook or Google login to work with your app. Thankfully, most popular login providers have standardized around some version of [OAuth](https://en.wikipedia.org/wiki/OAuth), and Meteor supports some of the most popular login services out of the box.
+
+<h3 id="supported-login-services">Facebook, Google, and more</h3>
+
+Here's a complete list of login providers for which Meteor provides core packages:
+
+1. Facebook with `accounts-facebook`
+2. Google with `accounts-google`
+3. GitHub with `accounts-github`
+4. Twitter with `accounts-twitter`
+5. Meetup with `accounts-meetup`
+6. Meteor Developer Accounts with `accounts-meteor-developer`
+
+There is a core package for logging in with Weibo, but it is no longer being actively maintained.
+
+<h3 id="oauth-logging-in">Logging in</h3>
+
+If you are using an off-the-shelf login UI like `accounts-ui` or `useraccounts`, you don't need to write any code after adding the relevant package from the list above. If you are building a login experience from scratch, you can log in programmatically using the [`Meteor.loginWith<Service>`](http://docs.meteor.com/#/full/meteor_loginwithexternalservice) function. It looks like this:
+
+```js
+Meteor.loginWithFacebook({
+  requestPermissions: ['user_friends', 'public_profile', 'email']
+}, (err) => {
+  if (err) {
+    // handle error
+  } else {
+    // successful login!
+  }
+});
+```
+
+<h3 id="oauth-configuration">Configuring OAuth</h3>
+
+There are a few points to know about configuring OAuth login:
+
+1. **Client ID and secret.** It's best to keep your OAuth secret keys outside of your source code, and pass them in is through Meteor.settings. Read how in the [Security article](security.html#api-keys-oauth).
```

Comment:
typo?
and pass them in is through Meteor.settings -> and pass them in through Meteor.settings